lib/sumup.dart

```diff
@@ -101,6 +101,21 @@ class Sumup {
     return SumupPluginResponse.fromMap(method);
   }
 
+  /// To prepare a SumUp card terminal for checkout, a registered SumUp merchant
+  /// account needs to be logged in and the card terminal will have been already
+  /// setup. Calling prepareForCheckout() before instancing a checkout will
+  /// speed up the checkout time.
```

Comment:
I'd make this doc comment a bit simpler and I'd keep the 'warning' that you can find in `wakeUpTerminal`'s doc comment, i.e. 'Don't call this method during checkout because it can lead to checkout failure'. Or something similar